### DISCUSSION TITLE
Depleted Fuel SMG Magazine is not recognized as an ingredient in its own recycle recipe.

### DISCUSSION THREAD

**Discussion by Sarynkia** [FakeFishGames/Barotrauma#14501] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Disclaimer for the disclaimer: only texture mods were in use.

As titled. The recycling recipe for the Depleted Fuel SMG Magazine does not recognize a spent DF SMG Magazine as an ingredient - only "standard" SMG Magazines can be used in the recipe.

The reverse case (crafting a standard SMG Magazine from a spent DF one) is not tested.

### Reproduction steps

1. Get a Depleted Fuel SMG Magazine and the necessary talents to recycle/craft it.
2. Use up the magazine.
3. Try to recycle it with only the DF SMG Magazine in the inventory.
4. Verify that it cannot be recycled.
5. Discard the spent DF SMG Magazine.
6. Obtain a "standard" SMG magazine and use it up the same way.
7. Again try to craft a DF SMG Magazine through recycling.
8. Verify that recycling is now available.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I unfortunately could not reproduce the issue, recycling the magazine seems to work with no issues at my end:
https://github.com/user-attachments/assets/f6306245-24a5-4bfb-b0a1-b736815fd16b

You marked that you're using mods, I wonder is some of those could be modifying the fabrication recipes in some way for example?

**Comment by Sarynkia** [comment]
Hi Regalis,
I have made further tests and you're right, it is indeed the mod that's causing problem. I'll report this to the mod author.

Apologies for the inconvenience.
